### ISSUE TITLE
Delete server

### ISSUE THREAD

**Issue by lolBoi9696** [Minehut/Meta#521]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [X] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [X] This issue does not contain any sensitive information 

**Platform**

Both

**Describe the feature**

I want to be able to delete servers because people are asking "how do you delete a server" and it's kinda annoying

**How the feature is useful**

People won't have to ask how to delete a server everytime and then the kids and things or other people don't have to say

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by LukasmanMHdude** [comment]
TBH not worth it, I'd assume they would code it if it was easy. You can alr hit the reset button and change the name of it so.

**Comment by LukasmanMHdude** [comment]
idk would like to here @CoreyShupe 's opinion on this

**Comment by lolBoi9696** [comment]
Ik @LukasmanMHdude it's just it's kinda annoying when people ask you how to delete my server and I kinda like a delete server somewhere in danger zone where people will find it easily and then people don't have to tell people that you can't delete your server

**Comment by DaRealMinecraftexpert** [comment]
If one would think it would be a Waste, could it probably Refund the Server Slot, and if it has one, the Icon for it? (Or potentially move the Icon to another server)

**Comment by MrTibo1** [comment]
> If one would think it would be a Waste, could it probably Refund the Server Slot, and if it has one, the Icon for it? (Or potentially move the Icon to another server)

It could just delete the entire server and open up the slot for if you want to make one in the future. I don't think it should refund anything

**Comment by DaRealMinecraftexpert** [comment]
You mean making the next purchaced slot free?

**Comment by MrTibo1** [comment]
> You mean making the next purchaced slot free?

Yes and no
This is already a thing anyway
You can buy slots in the shop and then use the **Create new server** button until you run out of slots and have to buy extra. It could just open up one of your slots again so you can create a new one later without losing the slot

**Comment by awaffle1** [comment]
This seems pointless. Just reset your server and change the name.

**Comment by lolBoi9696** [comment]
@awaffle1 
this feature is for the community not just me some people actually want to delete there server it would be a good option if there was a delete option needless to say it is kinda pointless but its just nice to have a thing like this

**Comment by awaffle1** [comment]
> @awaffle1
> this feature is for the community not just me some people actually want to delete there server it would be a good option if there was a delete option needless to say it is kinda pointless but its just nice to have a thing like this

"It's pointless but it would be nice" isn't a great reason, but this could still happen. I do see the point, but if even the author of the issue is saying the feature would be pointless it wouldn't be a bad idea to put this at the end of the line (if it even gets into the line at all).

**Comment by Sunni05** [comment]
Idk about you but I like the creation wizard lol

**Comment by MrScopes** [comment]
Yeah I’ve been asking for this feature for a while, it should just give you the server slot and not refund any icons. Or, if it refunds icons it could be half credits, or they could let you add the icon to another server.

**Comment by Sniped** [comment]
Personally, I think this feature isn't needed, you can simply just reset the server and change the name of it.

**Comment by lolBoi9696** [comment]
> Personally, I think this feature isn't needed, you can simply just reset the server and change the name of it.

same bro but i mean it a feature some people did want it and i mean people would like to delete there server completely sometimes

**Comment by OriginallySmokey** [comment]
Seems a little dangerous of an option.

**Comment by quick007** [comment]
Theres already "reset server", so how is this any more or less dangerous (assuming you'll be able to move the icons and whatnot to another server)
